cyctl/internal/get/modules.go

```diff
@@ -0,0 +1,37 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func ListModules(clientset *kubernetes.Clientset) {
+	labelSelector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"app.kubernetes.io/managed-by": "cyclops",
+		},
+	}
+
+	modules, err := clientset.AppsV1().Deployments("").List(context.TODO(), metav1.ListOptions{
+		LabelSelector: metav1.FormatLabelSelector(&labelSelector),
+	})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			fmt.Println("No modules found.")
+			return
+		}
+		fmt.Printf("Error listing modules: %v\n", err)
+		return
+	}
+
+	fmt.Println("NAME       AGE")
```

Comment:
I think we need a better way of formatting those columns. For longer deployment names, it's going to look messy.
<img width="496" alt="Screenshot 2024-04-21 at 11 13 46" src="https://github.com/cyclops-ui/cyclops/assets/72022639/0a3ada59-0ffe-4445-a6d9-5d4aeb4b2c9a">

Reply:
You can check this lib out https://github.com/ryanuber/columnize